tests/test_layout_objects.py

```diff
@@ -274,6 +275,59 @@ def test_accordion_active_false_not_rendered(self, settings):
             == 0
         )
 
+    def test_accordion_flush(self, settings):
+        test_form = SampleForm()
+        test_form.helper = FormHelper()
+        test_form.helper.layout = Layout(
+            Accordion(
+                AccordionGroup("one", "first_name")
+                # there is no ``flush`` kwarg here.
+            )
+        )
+
+        html = render_crispy_form(test_form)
+
+        # Since the default is False, there shouldn't be an accordion-flush class now.
+        assert html.count('<div class="accordion accordion-flush" id="accordion-') == 0
+
+        test_form.helper.layout = Layout(
+            Accordion(
+                AccordionGroup("one", "first_name"), flush=True
+            )  # now ``flush`` manually set as True
+        )
+
+        # We expect the class to be present now.
+        html = render_crispy_form(test_form)
+        assert html.count('<div class="accordion accordion-flush" id="accordion-') == 1
+
+    def test_accordion_always_open(self, settings):
+        test_form = SampleForm()
+        test_form.helper = FormHelper()
+        test_form.helper.layout = Layout(
+            Accordion(
+                AccordionGroup("one", "first_name"),
+                AccordionGroup("two", "last_name")
+                # No always_open kwargs. Default is False.
+            )
+        )
+
+        html = render_crispy_form(test_form)
+
+        # Since the default is False, there should two accounts for data-bs-parent
+        assert html.count('data-bs-parent="#accordion-') == 2
+
+        test_form.helper.layout = Layout(
+            Accordion(
+                AccordionGroup("one", "first_name", always_open=True),
+                AccordionGroup("two", "last_name", always_open=True)
```

Comment:
I agree it's a more DRY solution, but I tried to follow the behavior suggested by Bootstrap. It's similar to the active kwarg. One can set a single panel to be always open, for instance. If we put on the Accordion, it's all or nothing. But if you feel it's still better on the accordion, I'll make the necessary changes

Reply:
Is there a use case for setting this on only some of the items? Wouldn't you want this to be an "all or nothing" setting? 🤔

Reply:
I'm not sure. I personally can't imagine myself using this on just a few panels, but I have to imagine someone would. If a user decides to write all their HTML by hand, this would be a possibility.

In contrast, accordion-flush is all or nothing from the HTML. You can't flush a single panel. But you can "always open" a single panel (for whatever reason), since it's a panel behavior, not an accordion behavior.

So I guess we're down to deciding if we're going to favour ease of use or fidelity to the framework's (bootstrap) intended use.
I'm inclined for going the ease of use case, as you suggested. Just let me know and I'll work on the changes ✌️

Reply:
Thank you for your thoughts. 

I think lets's go with the approach that is easier to use and which meets the most common need. I don't think that we should aim to support & document two ways of doing the same thing.

Reply:
Great. I'll make the necessary changes later today.
Do you think it's better to close this PR and open another with less commits?

Reply:
It's ok. I'll squash and merge when we're ready.

Reply:
Alright, thanks! Always_open is now a BS5Accordion arg